src/utils/display.ts

```diff
@@ -37,16 +37,13 @@ const makeFinalContent = (
 };
 
 export const makeSignCanonAddrRegistryEntryContent = (
-  entry: CanonAddrRegistryEntry,
-  chainId: bigint,
-  registryAddress: Address
+  entry: CanonAddrRegistryEntry
 ): Panel => {
   const heading = "Confirm signature to register canonical address";
   const messages = [
-    `Ethereum Address: ${entry.ethAddress}`,
-    `Nocturne Canonical Address Nonce: ${entry.perCanonAddrNonce}`,
-    `Chain id: ${chainId}`,
-    `Registry address: ${registryAddress}`,
+    `Registering your canonical address provisions your account with an address with which you
+      receive private payments.`,
+    `Your connected Ethereum address: **${entry.ethAddress}**`,
```

Comment:
This definitely makes it more clear.

Suggestion: say `gives your Ethereum account ...` instead of `provisions your account ...`. Reasons:
1. `provisions` is somewhat "sophistocated" vocab, which sounds a lot more "professional" / "trustworthy", but it might not be as readable to people whose native tounge isn't English.
2. there's some ambiguity on "which account I'm upgrading" with the ability to receive private payments, so explicitly saying "Ethereum account" may help